kairon/api/app/routers/bot/bot.py

```
@@ -1658,3 +1660,60 @@ async def get_slot_actions(
+        {key: value for key, value in item.to_mongo().items() if key not in {"_id", "user"}}
+        for item in data
+    ]
+
+    return {"data": formatted_data}
+
+
+
+@router.post("/mail/config", response_model=Response)
+async def set_mail_config(
+        request_data: MailConfigRequest,
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS)
```

Ruff

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)